library/src/jit_library/include/device_gemm_multiple_d.hpp

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2022, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <cstdlib>
+#include <vector>
+#include <memory>
+#include <sstream>
+#include <iterator>
+#include <numeric>
+#include "ck/solution_instances/gemm_add_add_fastgelu_instances.hpp"
+#include "ck/ck.hpp"
+#include "ck/utility/math.hpp"
+#include "ck_headers.hpp"
+
+
+namespace ck {
+namespace tensor_operation {
+namespace device {
+namespace device_gemm_multiple_d {
+
+
+struct Solution
+{
+    std::string template_str;
+    index_t block_size;
+    index_t grid_size;
+
+    Solution(std::string s, index_t b, index_t g) : template_str(s), block_size(b), grid_size(g)
```

Comment:
Since these are public members, just remove the constructor and member functions. We can access the information directly with `solution.block_size`.